kafka_consumer_freshness_tracker/src/test/java/com/tesla/data/consumer/freshness/ConsumerFreshnessTest.java

```diff
@@ -272,6 +277,137 @@ public void testBurrowMissingConsumerGroupPartitionEndOffsetMarksMissing() throw
     });
   }
 
+  @Test
+  public void testConfigurationContainsUnknownCluster() throws Exception {
+    Burrow burrow = mock(Burrow.class);
+    String clusterName = "bad_cluster";
+    when(burrow.getClusterDetail(clusterName)).thenThrow(new IOException());
+    Map<String, Object> conf = mockConfForCluster(clusterName, "l1.example.com:9092",
+            "l2.example.com:9092", "l3.example.com:9092");
+
+    withExecutor(executor -> {
```

Comment:
why are these explicitly using the executor? It seems like we can test without running in a threadpool

Reply:
done